XML/syntax_test_XML.xml

```diff
@@ -0,0 +1,14 @@
+<!-- SYNTAX TEST "Packages/XML/XML.sublime-syntax" -->
+<?xml version="1.0" encoding="UTF-8"?>
+<!--                                ^ punctuation.definition.tag.end.xml -->
+
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--                                                                                                 ^ punctuation.definition.tag.end.xml -->
+
+<tagname attr="stuff">Contents</tagname>
+<!-- ^ text.xml meta.tag.xml entity.name.tag.localname.xml  -->
+<!--      ^ text.xml meta.tag.xml entity.other.attribute-name.localname.xml -->
+
+<таĝñäᴹə ατţř="șƬűʃ⨍">Contents</таĝñäᴹə>
```

Comment:
@keith-hall That character appears to be http://www.fileformat.info/info/unicode/char/1d39/index.htm, which according to https://www.w3.org/TR/REC-xml/#d0e804 is valid both as the start of a name, and in the less restrictive case of part of a name that is not the first character.

Either way, I think handling of unicode XML tag names is going to depend on the library/software used. Some characters may be supported by some, and not by others, but we should highlight anything that has a semblance of being valid.